src/wallet/rpcwallet.cpp

```diff
@@ -4749,6 +4798,7 @@ static const CRPCCommand commands[] =
     { "wallet",             "upgradewallet",            &upgradewallet,            true,  {} },
     { "wallet",             "sethdseed",                &sethdseed,                true,  {"newkeypool","seed"} },
     { "wallet",             "getnewaddress",            &getnewaddress,            true,  {"label"} },
+    { "hidden",             "getnewexchangeaddress",    &getnewexchangeaddress,    true,  {"label"} },
```

Comment:
why is the new rpc hidden?

Reply:
Its technically not for the normal user but an exchange so access for everyone not necessarily needed